platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -533,18 +533,14 @@ namespace platoon_strategic_ihp
                 status_msg.predecessor_position = pm_.getPredecessorPosition();
 
                 // Note: use isCreateGap to adjust the desired gap send to control plugin 
-                double regular_gap = std::max(config_.standStillHeadway, desired_headway);
+                double regular_gap = status_msg.desired_gap;
```

Comment:
I remember us having a safety concern with this as a safety issue. Can we guarantee this field is populated?

Reply:
The important gap information are calculated above at line 524 ("status_msg.desired_gap") which is always populated.
The regular gap is just a local variable and never used, since it is only used for cut-in to the middle.